dft-d3/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, gfortran }:
+stdenv.mkDerivation rec {
+    pname = "dft-d3";
+    version = "3.2rev0";
+
+    nativeBuildInputs = [ gfortran ];
+
+    src = fetchurl {
+      url = "https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dft-d3/dftd3.tgz";
+      sha256 = "0n8gi0raz8rik1rkd4ifq43wxs56ykxlhm22hpyq3ak1ixszjz6r";
+    };
+
+    unpackPhase = ''
```

Comment:
Is this really needed? A `tgz` should be handled by the default `unpackPhase`?

Reply:
Unfortunately not. `unpackPhase produced no directory` or something like this happens.